manifests/i/IniTranslator/IniTranslator/1.9.0.52/IniTranslator.IniTranslator.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: IniTranslator.IniTranslator
+PackageVersion: 1.9.0.52
+PackageLocale: en-US
+Publisher: Peter Thörnqvist
+PackageName: IniTranslator

```

```suggestion
PackageName: Ini Translator
```
This must match Control Panel.